libs/tup-components/src/projects/ProjectsLayout.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { Section } from '@tacc/core-components';
-import { ProjectsSummaryListing } from './ProjectsSummaryListing';
+import { ProjectsSummaryListing } from '../projects';
+import { AllocationsTable } from '../allocations';
 
 const ProjectsLayout: React.FC = () => {
   return (
-    <Section
-      header="Projects & Allocations"
-      content={<ProjectsSummaryListing />}
-    />
+    <Section header="Projects & Allocations">
+      <div>
+        <ProjectsSummaryListing />
+        <AllocationsTable />
+      </div>
```

Comment:
**Extra Markup:** Please avoid wrapper divs unless the are necessary. An empty `<div>` is usually a sure isgn of excess markup, but even one with a class might be excess. We'll learn as we go along.

```suggestion
      <ProjectsSummaryListing />
      <AllocationsTable />
```